server/controllers/finance/reports/operating/index.js

```diff
@@ -25,9 +26,9 @@ const DECIMAL_PRECISION = 2; // ex: 12.4567 => 12.46
  * This function does exactly the same thing except the value of amount will be
  * debit - credit this will know the expense account
  */
-function getQueryExpense() {
+function getQueryExpense(rate = 1) {
```

Comment:
It looks like this is the same code as `getAccountBalancesByTypeId`.  Could we just delete this code and replace it with the `getAccountBalancesByTypeId` call?